python/rmm/__init__.py

```diff
@@ -17,28 +17,22 @@
 from rmm.mr import disable_logging, enable_logging, get_log_filenames
 from rmm.rmm import (
     RMMError,
-    RMMNumbaManager,
-    _numba_memory_manager,
     is_initialized,
     register_reinitialize_hook,
     reinitialize,
-    rmm_cupy_allocator,
-    rmm_torch_allocator,
     unregister_reinitialize_hook,
 )
 
 __all__ = [
     "DeviceBuffer",
     "RMMError",
-    "RMMNumbaManager",
     "disable_logging",
     "enable_logging",
     "get_log_filenames",
     "is_initialized",
     "mr",
     "register_reinitialize_hook",
     "reinitialize",
-    "rmm_cupy_allocator",
     "unregister_reinitialize_hook",
 ]
 
```

Comment:
To make the transition easier for users, should we define a `__getattr__` here that prints a useful warning about where to now find the allocators, if someone attempts to import them?

Reply:
> To make the transition easier for users, should we define a `__getattr__` here that prints a useful warning about where to now find the allocators, if someone attempts to import them?

Good idea, done as a FutureWarning. So we can remove in 23.06.